src/content/nl/2020/third-parties.md

```diff
@@ -0,0 +1,298 @@
+
+{{ figure_markup(
+  image="requests-before-dom-by-category.png",
+  caption="Vroege verzoeken van derden per categorie.",
+  description="Kolomdiagram met het percentage verzoeken dat is geladen voordat DOM-inhoud is geladen. Openbare CDN-bronnen zijn hoogstwaarschijnlijk 50% op desktop, terwijl advertentiebronnen het minst waarschijnlijk zijn 7%",
+  width=600,
+  height=371,
+  sheets_gid="2118409936",
+  sql_file="percent_of_third_party_loaded_before_DOMContentLoaded.sql"
+  )
+}}
+
+## CPU-impact
+
+Niet alle bytes op het web zijn gelijk: een afbeelding van 500KB is misschien veel gemakkelijker voor een browser om te verwerken dan een gecomprimeerde JavaScript-bundel van 500KB, die oploopt tot 1,8MB aan client-side code! De impact van scripts van derden op de CPU-tijd kan veel kritischer zijn dan de extra bytes of tijd die op het netwerk wordt doorgebracht.
+
+We kunnen de aanwezigheid van categorieën van derden correleren met de totale CPU-tijd op de pagina, dit stelt ons in staat om de impact van elke categorie van derden op de CPU-tijd te schatten.
+
+{{ figure_markup(
+  image="cpu-time-by-category.png",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSNXusoFJKi8Z1i_yuG_5umUCJOZwBMaZaECq4T8lU5zIQuLR7UHfZOJmXsXzPWQTJFnFa3dcOEPJgy/pubchart?oid=225817673&format=interactive",
+  caption="Verdeling van CPU-tijd per categorie.",
+  description="Verdeling van de CPU-laadtijd door de aanwezigheid van categorieën van derden. De meeste categorieën volgen hetzelfde patroon, waarbij reclame voor de uitbijter een hogere CPU-laadtijd laat zien, vooral bij lagere percentielen.",
+  width=600,
+  height=371,
+  sheets_gid="727028027",
+  sql_file="distribution_of_size_and_time_by_third_parties.sql"
+  )
+}}
+
+Deze grafiek toont de kansdichtheidsfunctie van de totale CPU-tijd van de pagina door de categorieën van derden die op elke pagina aanwezig zijn. De mediaanpagina staat op 50 op de percentielas. De gegevens laten zien dat alle categorieën van derden een vergelijkbaar patroon volgen, met een gemiddelde pagina tussen de 400 - 1.000 ms CPU-tijd. De uitbijter hier is adverteren (in het zwart): als een pagina reclametags heeft, is de kans veel groter dat het CPU-gebruik tijdens het laden van de pagina wordt verhoogd. De mediaanpagina met reclametags heeft een CPU-laadtijd van 1.500 ms, vergeleken met 500 ms voor pagina's zonder reclame. De hoge CPU-laadtijd bij de lagere percentielen geeft aan dat zelfs de snelste sites aanzienlijk worden beïnvloed door de aanwezigheid van derden die als advertenties worden gecategoriseerd.
+
+## Prevalentie van Timing-Allow-Origin
```

Comment:
This should be in code format and lowercase. Can we change this in English too? And Japanese if you don't mind?

```suggestion
## Prevalentie van `timing-allow-origin`
```